tests/integration/test_flatten.py

```diff
@@ -67,11 +35,33 @@ def run_test(np_arr, num_arr):
 
 
 # test ndarray.flatten w/ 1D, 2D and 3D arrays
+@pytest.mark.parametrize("order", ["C", "F", "A"], ids=str)
```

Comment:
I don't think `ids=str` is necessary, since the values are all already strings. Also also as always, prefer immutable tuples wherever possible
```suggestion
@pytest.mark.parametrize("order", ("C", "F", "A"))
```

Reply:
Fixed. Thanks.